pkg/vm/builtins/modulo.go

```diff
@@ -320,27 +321,32 @@ func (m *ModBuiltin) writeNWordsValue(mem *memory.Memory, addr memory.MemoryAddr
 // Returns true on success or if all values are already known.
```

Comment:
Shouldn't this comment change now that you changed the return value?

Reply:
I see you are returning different possible values, it'd be nice to explain what do they mean